src/http/Stream.h

```diff
@@ -9,8 +9,14 @@
 #ifndef SQUID_SRC_HTTP_STREAM_H
 #define SQUID_SRC_HTTP_STREAM_H
 
+#include "clientStreamForward.h"
```

Comment:
Please remove the old clientStreamNode and ClientHttpRequest forward declarations below if we have to add this header.

Reply:
ok, done